Game_Development/Rock-Paper-Scissors/RPS.py

```diff
@@ -1,48 +1,105 @@
+import tkinter as tk
+from tkinter import PhotoImage
 import random
-import math
+from PIL import Image, ImageTk  # Pillow library for image handling

```

Comment:
Hey @icemberg ,
Please mention in README file required libraries and the versions as it is empty. 
Also please work on the rendering of the game, will approve after the changes.
<img width="497" alt="Screenshot 2024-11-08 at 5 52 03 AM" src="https://github.com/user-attachments/assets/056ef170-5511-442f-94db-6ec90bb64fc0">

Thank you!